nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/pom.xml

```diff
@@ -83,7 +83,6 @@
         <dependency>
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-distributed-cache-client-service-api</artifactId>
-            <scope>test</scope>
```

Comment:
Is there a reason that this previously was marked 'test' scope and now is being changed to compile scope?

Reply:
I dropped the test scope after I see integration tests fail without this dependency.

> mvn clean package verify -Pintegration-tests

Reply:
For this review, I reverted back.